src/connectors/voice-search/connectVoiceSearch.ts

```diff
@@ -0,0 +1,105 @@
+import noop from 'lodash/noop';
+import {
+  checkRendering,
+  createDocumentationMessageGenerator,
+} from '../../lib/utils';
+import { Renderer, RenderOptions, WidgetFactory } from '../../types';
+import voiceSearchHelper from '../../lib/voiceSearchHelper';
+import { VoiceListeningState } from '../../widgets/voice-search/voice-search';
+
+const withUsage = createDocumentationMessageGenerator({
+  name: 'voice-search',
+  connector: true,
+});
+
+export interface VoiceSearchRenderOptions extends RenderOptions {
+  isSupportedBrowser: () => boolean;
```

Comment:
Why isn't it a simple boolean?

Reply:
VoiceSearchHelper has a function with same name, but here it doesn't have to be a function. So I've changed it to a boolean variable.